client/jetpack-cloud/sections/agency-dashboard/sites-overview/site-status-content/index.tsx

```diff
@@ -177,6 +177,10 @@ export default function SiteStatusContent( {
 						isLargeScreen={ isLargeScreen }
 						item={ rows }
 						siteError={ hasAnyError }
+						disabled={ rows.site.value.is_atomic }
```

Comment:
Can we use the useRowMetadata hook to do this and the tooltip? We are already handling a few use cases for atomic sites here. I feel like having a centralized place makes it easier to maintain.

Reply:
I agree with what you said. We can do this on a separate PR as there are a couple of areas we need to refactor that are beyond this PR's scope (e.g., Site name, tooltips, etc.)

Reply:
Sure! Could we please create an issue in our maintenance board?

Reply:
Done 👍 https://github.com/orgs/Automattic/projects/757/views/1?pane=issue&itemId=42010209